Channels/UserInterface/02c_SemiAutomaticMapping.md

```diff
@@ -0,0 +1,94 @@
+        Click this entry to rename the selected view. The *Rename view* window is displayed. This menu entry is only displayed if a view has been selected.
+
+      - ![Reset](../../Assets/Icons/Reset.png "[Reset]") reset    
+        Click this entry to reset all unsaved changes to the settings of the selected view. This menu entry is only displayed if a view has been selected and any changes have been made to the view settings.
+
+      - ![Publish](../../Assets/Icons/Publish.png "[Publish]") publish    
+        Click this entry to publish the view. This menu entry is only displayed if a view has been selected and unpublished.
+
+      - ![Private view](../../Assets/Icons/Unpublish.png "[Unpublish]") private view    
+        Click this entry to unpublish the view. This menu entry is only displayed if a view has been selected and published.
+
+      - ![Save](../../Assets/Icons/Save.png "[Save]") save    
+        Click this entry to save the current view settings in the selected view. This menu entry is only displayed if a view has been selected.
+
+        > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.
+
+      - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete    
+        Click this entry to delete the selected view. A confirmation window is displayed to confirm the deletion. This menu entry is only displayed if a view has been selected.
+
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *List changes that cannot be confirmed by you*    
+  Enable this toggle to list all offers including those that cannot be confirmed by you. Disable the toggle to list only offers that can be confirmed by you. By default, this toggle is disabled.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all offers in the list are selected.
+
+- ![Lock](../../Assets/Icons/Lock01.png "[Lock]") (Lock)   
+  Locked offers cannot be confirmed by you. These offers are greyed out and cannot be selected.
+    
+- [CONFIRM]    
+  Click this button to confirm the changes in the selected offer(s). The offer is uploaded with the confirmed changes and removed from the list. 
+
+- [REJECT]   
+  Click this button to reject the changes in the selected offer(s). The offer is removed from the list. 
+
+Depending on whether the *List changes that cannot be confirmed by you* drop-down list is enabled or not, the list displays all offers including those that cannot be confirmed by you or only offer that can be confirmed by you. Depending on the settings, the displayed columns may vary. All fields are read-only. 
```

Comment:
Ansprache an Benutzer: "by you" oder "by the current user/(currently) logged in user?